nixos/modules/services/monitoring/prometheus/exporters/smartctl.nix

```diff
@@ -60,7 +51,7 @@ in {
         ]
       );
       ExecStart = ''
-        ${pkgs.prometheus-smartctl-exporter}/bin/smartctl_exporter -config ${configFile}
+        ${pkgs.prometheus-smartctl-exporter}/bin/smartctl_exporter --web.listen-address="${cfg.listenAddress}:${toString cfg.port}" --smartctl.path="${pkgs.smartmontools}/bin/smartctl" --smartctl.interval="${cfg.maxInterval}" ${deviceArgs}
```

Comment:
We could probably shorten this by using `systemd.services.<name.>path`.

Reply:
It defaults to `/usr/sbin/smartctl`, so, I think it'll not work https://github.com/prometheus-community/smartctl_exporter/blob/61e28093f6d93daefa71cc3c1b5869b4a07b6eda/main.go#L65

Reply:
I was wondering if it would pick it up from PATH when we just pass `--smartctl.path=smartctl`.

Reply:
Seems like it's impossible to set a path here b/c of the exporter module structure...